packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -19,26 +19,65 @@ import { parseArgs } from 'node:util';
  * @import {FastUSDCConfig} from '@agoric/fast-usdc/src/fast-usdc.start.js'
  */
 
+const { keys } = Object;
+
+/** @type {Record<string, Pick<FastUSDCConfig, 'oracles'>>} */
+const configurations = {
+  A3P_INTEGRATION: {
+    oracles: {
+      gov1: 'agoric1ee9hr0jyrxhy999y755mp862ljgycmwyp4pl7q',
```

Comment:
Just wondering, where do all these accounts come from? I'm guessing we have the private keys somewhere so that a script can use them? I recognize gov1 and gov2 from emerynet at least...

Reply:
I have the same question. please include a comment on how where these values come from and when they'd need updating

Reply:
https://github.com/Agoric/agoric-3-proposals/blob/main/packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh

Reply:
oh... right... the OCW operator addresses are not necessarily the same as the oracle operators.